main.go

```diff
@@ -36,7 +36,8 @@ var validParsers = []string{
 
 // GlobalOptions has all the top level CLI flags that honeytail supports
 type GlobalOptions struct {
-	APIHost string `hidden:"true" long:"api_host" description:"Host for the Honeycomb API" default:"https://api.honeycomb.io/"`
+	APIHost    string `hidden:"true" long:"api_host" description:"Host for the Honeycomb API" default:"https://api.honeycomb.io/"`
+	TailSample bool   `hidden:"true" long:"tail_sample" description:"When true, sample while tailing. When false, sample post-parser events"`
```

Comment:
Sticking this in options, but the `hidden:true` effectively means it's not user-facing. It won't show up in --help so the only way to set it would be if you knew it existed. 
I put it in options because that seemed the most reasonable way to pass it around along with all the other bits. 
hmm.  Removing the long:"tail_sample" bit might help it be even more non-user-settable. I'll do that.